app/client/cypress/integration/home/links.spec.js

```diff
@@ -17,4 +17,28 @@ describe('Homepage', () => {
     cy.findByText('National').click();
     cy.url().should('include', '/national');
   });
+  it(`"Data" button links to the data page`, () => {
+    cy.findByText('Data').click();
+    cy.url().should('include', '/data');
+  });
+  it(`"About" button links to the about page`, () => {
```

Comment:
Add blank line between tests.
```suggestion
  });
  
  it(`"About" button links to the about page`, () => {
```